seeds/tables/projects.js

```diff
@@ -1,17 +1,30 @@
-const { sample } = require('lodash');
+const { sample, sampleSize } = require('lodash');
 const projects = require('../data/projects.json');
 
 module.exports = {
   populate: knex => {
     return Promise.all(
       projects.map(project => {
+        return knex('profiles').then(profiles => {
+          return knex('projects').insert({
+            licenceHolderId: sample(profiles).id,
+            ...project
+          });
+        });
+      })
+    );
+  },
+  populateList: knex => {
+    const projectsList = sampleSize(projects, 5);
+    return Promise.all(
+      projectsList.map(project => {
         return knex('profiles')
-          .then(profiles => {
-            return knex('projects')
-              .insert({
-                licenceHolderId: sample(profiles).id,
-                ...project
-              });
+          .first()
+          .then(profile => {
+            return knex('projects').insert({
+              licenceHolderId: profile.id,
+              ...project
+            });
```

Comment:
I guess this will result in adding the projects list to random profiles, then re-adding 5 duplicate projects to the first profile returned by the DB. It could look a little funky as the same project could be added to the first profile twice, showing the same title, licenceNumber and expiryDate. We could avoid this by splitting the array of projects to make sure each one is only added once. But saying that I'm not sure we really care as its sample data :)

Reply:
Another thing you could do is just change the insert to an update and just change the licenceHolderId for the 5 that are assigned to the first profile